### PR TITLE
refactor: Replace PDF course certificate view code

### PR DESCRIPTION
**Pull request by ruzniaievdm**
https://github.com/raccoongang/edx-platform/pull/2423
[JIRA](https://openedx.atlassian.net/browse/DEPR-157)
[YT](https://youtrack.raccoongang.com/issue/OeX_Depr-12)

### CODE REVIEW

File: src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx [hunk 1]
```diff
@@ -81,8 +81,6 @@ function CertificateStatusAlert({ intl, payload }) {
       alertProps.header = intl.formatMessage(certMessages.certStatusDownloadableHeader);
       if (isWebCert) {
```
Comment (dyudyunov) [inline src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx:82]: Is there a way when `certStatus === CERT_STATUS_TYPE.DOWNLOADABLE` but the `isWebCert` is False?

I mean do we need this `if` statement?

**Review by dyudyunov — APPROVED** [review]

File: src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx (hunk 1 above)
Comment (ruzniaievdm) [reply src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx:82]: `isWebCert` - True if html web certificates are enabled and there is active web certificate
I thought about it, and rather you are right and the IF statement makes no sense

**Review by idegtiarov — CHANGES_REQUESTED** [review]
LGTM, the only question I have is why we've removed from the translation `courseCelebration.downloadButton` but there were no mentioned any parts in the code about courseCelebration?

**Comment by ruzniaievdm** [comment]
@idegtiarov `message.downloadButton` was removed from `src/courseware/course/course-exit/CourseCelebration.jsx` file after this translation is out of date

**Pull request closed without merging**